tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -105,21 +135,26 @@ package <file.genPackage> // <file.grammarName>
 package parser // <file.grammarName>
 <endif>
 
+<if(file.antlrRuntimeImport)>
+import "<file.antlrRuntimeImport>"
+<else>
 import "github.com/antlr/antlr4/runtime/Go/antlr"
+<endif>
 <if(header)>
 
 <header>
 <endif>
 
 // A complete Visitor for a parse tree produced by <file.parserName>.
 type <file.grammarName>Visitor interface {
-	antlr.ParseTreeVisitor
+    antlr.ParseTreeVisitor
+    <file.visitorNames:{lname | <lname; format="cap">ContextVisitor}; separator="\n">
```

Comment:
That's my bad, then.  I couldn't see how it was implementing a partial interface.  Sorry about that.  I'll try implementing a simple labeled expression evaluator in it, to see how it goes.